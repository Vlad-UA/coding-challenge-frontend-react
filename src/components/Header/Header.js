// Core
import React, { PureComponent } from 'react';

// Instruments
import styled from 'styled-components';
import logo from '../../assets/logo.png';

const Wrapper = styled.div`
  display: grid;
  grid-template-columns: 100px auto;
`;

const Img = styled.div`align-self: center;`;

export default class Header extends PureComponent {
  render() {
    return (
      <Wrapper>
        <Img>
          <img src={logo} alt="logo" />
        </Img>
        <div>
          <h1>Police Department of Berlin</h1>
          <h2>Stolen bikes</h2>
        </div>
      </Wrapper>
    );
  }
}
