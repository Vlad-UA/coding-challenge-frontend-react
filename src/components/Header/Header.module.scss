.Header{
  display: grid;
  grid-template-columns: 100px auto;

  img {
    align-self: center;
  }
}
