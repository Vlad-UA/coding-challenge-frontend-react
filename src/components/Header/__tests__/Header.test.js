/* eslint-env jest */

// Core
import React from 'react';

// Test utilities
import renderer from 'react-test-renderer';

// Component
import Header from '../Header';

describe('Component Header:', () => {
  test('Must match snapshot:', () => {
    const header = renderer
      .create(<Header />)
      .toJSON();

    expect(header).toMatchSnapshot();
  });
});
