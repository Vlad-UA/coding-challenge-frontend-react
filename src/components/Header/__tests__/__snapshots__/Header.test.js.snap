// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header: Must match snapshot: 1`] = `
<div
  className="Header"
>
  <img
    alt="logo"
    src="logo.png"
  />
  <div>
    <h1>
      Police Department of Berlin
    </h1>
    <h2>
      Stolen bikes
    </h2>
  </div>
</div>
`;
